dll/ime/msctfime/msctfime.c

```diff
@@ -0,0 +1,336 @@
+/*
+ * PROJECT:     ReactOS msctfime.ime
+ * LICENSE:     LGPL-2.1-or-later (https://spdx.org/licenses/LGPL-2.1-or-later)
+ * PURPOSE:     Supporting IME interface of Text Input Processors (TIPs)
+ * COPYRIGHT:   Copyright 2023 Katayama Hirofumi MZ <katayama.hirofumi.mz@gmail.com>
+ */
+
+#include "msctfime.h"
+
+WINE_DEFAULT_DEBUG_CHANNEL(msctfime);
+
+HINSTANCE g_hInst = NULL; /* The instance of this module */
+
+BOOL WINAPI
+ImeInquire(
```

Comment:
Since this is brand new ReactOS-specific code, I would suggest you add SAL annotations to all the functions.

Reply:
Added SAL annotations.